server/container/guice/blob-objectstorage-guice/src/main/java/org/apache/james/modules/objectstorage/ObjectStorageBlobConfiguration.java

```diff
@@ -165,6 +165,11 @@ public ReadyToBuild bucketPrefix(Optional<String> bucketPrefix) {
                 return this;
```

Comment:
`JAMES-2812 use random bucket prefix each test `

I guess some code is no longer needed?